.github/test-matrix.yaml

```diff
@@ -0,0 +1,65 @@
+# This file is used as the [matrix](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix)
+# key in the test job defined in `.github/workflows/ci.yml`
+# after being processed with `.github/bin/build-matrix-from-impacted.py`
+# to exclude modules without any changes.
+include:
```

Comment:
FYI see https://github.com/trinodb/trino/pull/10838 - might need to adjust this file accordingly.

Reply:
I had conflicts when rebasing and resolved them, this file is correct and up-to-date.